schema.graphql

```diff
@@ -0,0 +1,81 @@
+# This file was generated based on ".graphqlconfig". Do not edit manually.
```

Comment:
This file seems the same file as `src/schema.graphql`, is it necessary?

Reply:
Indeed. That was something brought in by IDE. I've removed it now.